### ISSUE TITLE
`particle identify` does not show prerelease version information `-rc.x`

### ISSUE THREAD

**Issue by technobly** [particle-iot/particle-cli#423]
`particle identify` doesn’t show prerelease version information `-rc.x`:
```
Your device id is 123412341234123412341234
Your IMEI is 1234123412341234
Your ICCID is 12341234123412341234
Your system firmware version is 0.8.0
```

But if you connect via USB Serial and send `v` it does:
```
system firmware version: 0.8.0-rc.4
```

**Comment by technobly** [comment]
https://github.com/particle-iot/particle-cli/blob/a2d15a77f4049655e70c019d0ce992c09bb794c2/src/cmd/serial.js#L1400

### Would we just change this ^ to:
```javascript
const matches = data.match(/system firmware version:\s+(.*)/); 
```

**Comment by monkbroc** [comment]
Good one. Thanks

**Comment by monkbroc** [comment]
Fixed in 1.31.0. Try it with `npm install -g https://github.com/particle-iot/particle-cli/releases/download/v1.31.0-rc.2/particle-cli-1.31.0-rc.2.tgz`
